README.rst

```diff
@@ -92,22 +92,19 @@ My email: amjith.r@gmail.com, Twitter: `@amjithr <http://twitter.com/amjithr>`_
 Detailed Installation Instructions:
 -----------------------------------
 
-OS X:
-=====
+macOS:
```

Comment:
Haha, I think at 10.12 they officially changed the name to macOS so it matches their other products. OS X doesn't refer to their current operating system anymore.

But, if you have 10.11 or below, it's still OS X -- confusing!

I'm up for whichever. Personally, I think a single name is better (less reading, more clear, etc.).